example-workspace/BUILD

```diff
@@ -26,21 +26,46 @@ cc_binary(
     deps = ["//foo:bar_cc_proto"],
 )
 
+# Example python binary that uses the dynamic python validation code.
+#
+# Exactly as example_cc above except different label. Example:
+#
+# bazel run //:example_py -- $(pwd)/valid.textproto
+
+py_binary(
+    name = "example_py",
+    srcs = ["example.py"],
+    main = "example.py",
+    srcs_version = "PY3",
+    deps = [
+        "@com_google_protobuf//:protobuf_python",
+        "@com_envoyproxy_protoc_gen_validate//python:validator_py",
+        "//foo:bar_py_proto",
+    ],
+    imports = [
+        "../com_envoyproxy_protoc_gen_validate/python",
+    ],
```

Comment:
Can we do this without adding `imports`? The import declaration in the .py file might get a little more verbose.

Reply:
Yes that should work. I added the import like this so that our import would match the non-bazel/pip package import.